opm/simulators/flow/GenericTracerModel_impl.hpp

```diff
@@ -488,7 +493,7 @@ linearSolveBatchwise_(const TracerMatrix& M, std::vector<TracerVector>& x, std::
             x[nrhs] = 0.0;
             Dune::InverseOperatorResult result;
             solver.apply(x[nrhs], b[nrhs], result);
-            converged = (converged && result.converged);
+            converged &= result.converged;
```

Comment:
This will fail if `result.converged` is ever changed to something other than `bool`, so I'd prefer if we kept the original statement even if it's more to type.

Reply:
i find it less expressive but sure.

Reply:
> i find it less expressive but sure.

Maybe, but it's definitely more correct.